app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
 <p>Appwrite supports multiple protocols for accessing the server, including <a href="/docs/rest">REST</a>, <a href="/docs/graphql">GraphQL</a>, and <a href="/docs/realtime">Realtime</a>.</p>
 
-<p>The REST API allows you to manipulate your Appwrite server through many endpoints, each representing a specific method of modification to a specific resource type.</p>
+<p>The REST API allows you to access your Appwrite server through HTTP requests, without the need for an SDK. Each endpoint in the API represents a specific operation on a specific resource.</p>
 
-<p>Requests and responses through the REST API follow a rigid structure - that is, they always expect the same structure for requests to a specific endpoint, and responses from that endpoint will also always have the same structure. </p>
+<h2><a href="/docs/rest#headers" id="headers">Headers</a></h2>
+<p>Appwrite's REST APIs expect certain headers to be attached to each request:</p>
 
-<p>The specific structures used for Appwrite resources and their related endpoints can be found in the <b>References</b> section for each service.</p>
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 300px">Header</td>
+        <td>Description</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>X-Appwrite-Project: [PROJECT-ID]</code></td>
+        <td>Required, provides the ID of your Appwrite Project to the REST API.</td>
+    </tr>
+    <tr>
+        <td><code>Content-type: application/json</code></td>
+        <td>Required, declares content type of the HTTP request.</td>
+    </tr>
+    <tr>
+        <td><code>X-Appwrite-key: [API-KEY]</code></td>
+        <td>Optional, used for authentication in server integrations. <b>Do not use API keys in client applications.</b></td>
+    </tr>
+    </tbody>
```

Comment:
Good catch. What do they do XD Do you mind giving me a hand?

Reply:
`X-Appwrite-Response-Format` is for backward compatibility. It allows older SDKs to tell the server to respond data in the requested format (ie. an SDK for 0.15.x should still be able to work connecting to a 1.0.x Appwrite server).

Typically, in web scenarios, the cookie is automatically sent back to the browser after a session is created and the front end automatically sends the cookie for requests to Appwrite. `X-Fallback-Cookies` is for cases where cookies aren't available because maybe the browser restricts it due to it being a 3rd party cookie. See [Custom Domains](https://appwrite.io/docs/custom-domains) docs:
![image](https://user-images.githubusercontent.com/1477010/202799438-2ddb192c-4502-4a21-be32-56ba15ff7b9f.png)